tests/cross_functional/scale/noobaa/test_delete_objects.py

```diff
@@ -190,3 +201,96 @@ def test_delete_objects(
         # stop the io running in the background
         event.set()
         io_thread.result()
+
+    @bugzilla("2279742")
+    @bugzilla("2279964")
+    @polarion_id("OCS-6097")
+    @polarion_id("OCS-6096")
```

Comment:
I don't think adding multiple polarion IDs to a test function is not supported by the polarion importer plugin. Can we combine these two tests?

Reply:
I have done these in the past. There are already many tests with multiple polarion ids

Reply:
> I have done these in the past. There are already many tests with multiple polarion ids
There is no restriction on adding multiple polarion ids to a test func in ocs-ci. However, I doubt if the test execution status gets updated for all those polarion ids in polarion. At least, it was not supported by the polarion importer plugin the last time I checked, although that was a long time ago. Did you check if the test results were being updated for all those decorated polarion ids tests in polarion?